push-sdk-swiftTests/AGDeviceRegistrationTests.swift

```diff
@@ -127,4 +127,59 @@ class AGDeviceRegistrationTests: XCTestCase {
         
         waitForExpectationsWithTimeout(10, handler: nil)
     }
+    
```

Comment:
probably the Analytics tests should go in their own test class ```AGPushAnalyticsTests``` instead of reusing the Registration